bc_obps/registration/models/facility_designated_operation_timeline.py

```diff
@@ -23,6 +35,7 @@ class FacilityDesignatedOperationTimeline(TimeStampedModel):
     class Meta:
         db_table_comment = "A table to connect facilities and operations"
         db_table = 'erc"."facility_designated_operation_timeline'
+        # brianna how is status set?
```

Comment:
Not really related to this PR, but this constraint looks like it checks that only one facility can have a null end date. It doesn't seem tied to status though (unless I'm missing something? a trigger somewhere?), so we could have multiple facilities showing a status of `Active` without throwing a db error

Reply:
you are correct @Sepehr-Sobhani ! And thank you for mocking me - I still haven't caught up on all the emails I missed while I was out sick.